data/remnant/remnant missions.txt

```diff
@@ -940,6 +940,143 @@ mission "Remnant: Learn Sign Follow Up"
 
 
 
+mission "Remnant: Follow-up on Terminus Researchers A"
+	name "Check up on Wormhole Researchers"
+	description "Return to Bounty to check up on a group of wormhole researchers you met some time ago."
+	minor
+	source
+		government Remnant
+	destination Bounty
+	to offer
+		has "Remnant: Learning Sign 2: done"
+		has "Terminus exploration: done"
+		not "Deep: Wormhole Research: done"
+		not "Remnant: Follow-up on Terminus Researchers B: done"
+		not "Remnant: Follow-up on Terminus Researchers B: active"
+		random < 40
+	on offer
+		conversation
+			`Looking around at the Remnant spaceport, it suddenly strikes you how amazing this is. You never would have imagined that you would end up here back when you earned your pilot's license. It seems so long ago that your biggest concern was deciding whether to pay off your loan by ferrying cargo, doing courier jobs, or escorting merchants. (and on your darker days - contemplating being a pirate).`
```

Comment:
Given that the player is more likely than not to have not thought about being a pirate, I wouldn't have this line here implying that they did.

Reply:
OK. Removed.

Reply:
Given that capturing pirate ships is basically being a privateer (aka legalized piracy), it isn't that far of a stretch. There's not a big leap between capturing pirates and capturing merchants. James even talks about capturing ships in the intro.

But either way, removed that line for now.